src/pages/Basket.jsx

```diff
@@ -27,7 +30,23 @@ const BorderLinearProgress = styled(LinearProgress)(({ theme }) => ({
   },
 }));
 
+const getDaysBetweenDates = (date1, date2) => {
+  const differenceInTime = date1 - date2;
+  const differenceInDays = Math.floor(differenceInTime / (1000 * 3600 * 24));
+
+  return differenceInDays;
+}
+
 const Basket = () => {
+  const id = useParams();
+
+  const { name, ownerId: { firstName }, description, goal, value, expirationDate, isPublic, createdAt, image } = useSelector(state => state.basket.basket);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(get_basket_by_id({ id: id.basket }))
```

Comment:
what is in id? Can you pls rename it, cause it looks so weird when u taking basket from id

Reply:
?